quinn-h3/src/tests/helpers.rs

```diff
@@ -1,12 +1,11 @@
 use std::{
     fs, io,
     net::{IpAddr, Ipv6Addr, SocketAddr},
-    sync::atomic::{AtomicU16, Ordering, AtomicBool},
+    sync::atomic::{AtomicBool, AtomicU16, Ordering},
```

Comment:
This belongs in the previous commit. (Tip: `git rebase -x "cargo fmt -- --check"`.)

Reply:
Awesome!